File: Raven.Abstractions/Data/RestoreRequest.cs
//-----------------------------------------------------------------------
// <copyright file="RestoreRequest.cs" company="Hibernating Rhinos LTD">
//     Copyright (c) Hibernating Rhinos LTD. All rights reserved.
// </copyright>
//-----------------------------------------------------------------------
using System;

namespace Raven.Abstractions.Data
{
	public class RestoreRequest
	{
        public string BackupLocation { get; set; }
        
        [Obsolete]
		public string RestoreLocation { get { return BackupLocation; } set { BackupLocation = value; } }

		public string DatabaseLocation { get; set; }
		public string DatabaseName { get; set; }

        public string JournalsLocation { get; set; }
        public string IndexesLocation { get; set; }
	    public bool Defrag { get; set; }
	}
}

File: Raven.Client.Java/src/main/java/net/ravendb/abstractions/data/RestoreRequest.java
package net.ravendb.abstractions.data;


public class RestoreRequest {

  private String backupLocation;

  private String databaseLocation;

  private String databaseName;

  private String journalsLocation;

  private String indexesLocation;

  private boolean defrag;

  public String getBackupLocation() {
    return backupLocation;
  }

  public void setBackupLocation(String backupLocation) {
    this.backupLocation = backupLocation;
  }

  public String getDatabaseLocation() {
    return databaseLocation;
  }

  public void setDatabaseLocation(String databaseLocation) {
    this.databaseLocation = databaseLocation;
  }

  public String getDatabaseName() {
    return databaseName;
  }

  public void setDatabaseName(String databaseName) {
    this.databaseName = databaseName;
  }

  public String getJournalsLocation() {
    return journalsLocation;
  }

  public void setJournalsLocation(String journalsLocation) {
    this.journalsLocation = journalsLocation;
  }

  public String getIndexesLocation() {
    return indexesLocation;
  }

  public void setIndexesLocation(String indexesLocation) {
    this.indexesLocation = indexesLocation;
  }

  public boolean isDefrag() {
    return defrag;
  }

  public void setDefrag(boolean defrag) {
    this.defrag = defrag;
  }

}
